rnd/nextgenautogpt/__main__.py

```diff
@@ -0,0 +1,39 @@
+import multiprocessing as mp
+from typing import Any
+
+import nextgenautogpt.manager.manager as mod_manager
+import nextgenautogpt.server.server as mod_server
+
+
+def main() -> None:
+    # Create queues/pipes for communication
+    server_to_manager: mp.Queue[Any] = mp.Queue()
+    manager_to_server: mp.Queue[Any] = mp.Queue()
+
+    # Create and start server process
+    server: mp.Process = mp.Process(
```

Comment:
Also shouldn't the queue be based on the event name not by who's the publisher and consumer? 

e.g: you will have pool of queue with different event names (so we can also allow pub-sub), This is common practice in pretty much all the message queue tech, so it might be easier to migrate later to other thing latter.

Reply:
The intent of this PR is to give a program sketch so we can all work together on it.